### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by wjakelee**
Add FAQ statement

## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/faq.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ sudo lsof -i :8888
 You can either stop the process using the port, or change the metrics port that your collector uses.
 See [Accessing the collector's metrics](#accessing-the-collectors-metrics) section above.
 
+#### When I change the name of the collector in the configuration file and redeploy I see a new collector in the Sumo UI
+
+Changing the collector's name will result in re-registering the collector with Sumo Logic. The original collector will turn "inactive" soon after the change and will allow you to delete the collector in the Sumo UI.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/faq.md:98]: @wjakelee , thank you so much for submitting this! Do these edits capture your point correctly?

```suggestion
#### When I change the name of the collector in the configuration file and redeploy, I see a new collector in the Sumo UI. Why is this happening?

A: When you change the collector's name and redeploy, it re-registers as a new collector in Sumo Logic. The original collector will become "inactive" shortly after the change, allowing you to delete it from the Sumo UI.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Looks great, Jake! Had a few proposed changes.

**Comment by jpipkin1** [comment]
@wjakelee, I suggest changing "Sumo UI" to "Sumo Logic UI". The "[Voice and tone](https://help.sumologic.com/docs/contributing/style-guide/#voice-and-tone)" section in our style guide says:
>Our brand guideline is to always refer to “Sumo Logic”, rather than Sumo.

File: docs/send-data/opentelemetry-collector/faq.md (hunk 1 above)
Comment (wjakelee) [reply docs/send-data/opentelemetry-collector/faq.md:98]: Looks great! Thank you!

**Review by jpipkin1 — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

**Pull request merged**